lib/action.rb

```diff
@@ -20,13 +22,17 @@ def initialize(config)
   def check_version
     if version_changed?
       state = 'success'
-      description = 'Version is changed'
+      description = 'Updated'
     else
       state = 'failure'
-      description = 'Branch version is not changed'
+      description = failed_status_description
     end
 
-    client.create_status(repo, head_commit, state, description: description, context: 'version check')
+    client.create_status(repo, head_commit, state, description: description, context: 'Gem Version')
+  end
+
+  def failed_status_description
+    "Update: #{file_path}".truncate(140)
```

Comment:
IMO, this could be a bit more verbose. `"Please update the version file #{file_path}"`

Reply:
I've purposely not made it verbose to reduce the chances it hits the 140 character limit